pkg/clickhouse/flashback.go

```diff
@@ -68,7 +68,7 @@ package clickhouse
 // 	if err != nil {
 // 		return err
 // 	}
-// 	detachedParentDir := filepath.Join(dataPath, "data", TablePathEncode(diff.BTable.Database), TablePathEncode(diff.BTable.Table), "detached")
+// 	detachedParentDir := filepath.Join(dataPath, "data", common.TablePathEncode(diff.BTable.Database), common.TablePathEncode(diff.BTable.Table), "detached")
```

Comment:
If you will figure out how flashback command works
and add test for flashback command to `./test/integration/integration_test.go`
it will great

in feature PR, if you found unused commented code part, feel free to remove it

Reply:
> If you will figure out how flashback command works and add test for flashback command to `./test/integration/integration_test.go` it will great
> 
> in feature PR, if you found unused commented code part, feel free to remove it

For the merging reason, I don't think flashback works for mergetree. Hashes can not work out.